vanilla-framework/vanilla-brochure-theme
# Install example-js in docs

## Done
Replaced the hard coded examples in the docs with example-js. Updated some docs with basic descriptions.

## QA
- Make sure you pull down [docs.vanillaframework.io repo](https://github.com/ubuntudesign/docs.vanillaframework.io/)
- Pull down this branch in a parent directory to the docs one
- cd into the docs.vanillaframework.io repo
- Replace `build-html` file content with the content below:
``` bash
documentation-builder --base-directory ../vanilla-brochure-theme/docs  \
                      --site-root '/en/'  \
                      --output-path .  \
                      --template-path template.html  \
                      --tag-manager-code 'GTM-K92JCQ'  \
                      --no-link-extensions \
                      --force
```
- Run `./build-html`
- Then run `caddy`
- Go to http://127.0.0.1:8543/en/
- Click through the docs and check example js is rendering all examples

## Details
Fixes https://github.com/vanilla-framework/vanilla-brochure-theme/issues/97 https://github.com/vanilla-framework/vanilla-brochure-theme/issues/98
